hybrid/decomposers.py

```diff
@@ -422,6 +428,99 @@ def next(self, state, **runopts):
 
         return state.updated(subproblem=subbqm)
 
+class SublatticeDecomposer(traits.ProblemDecomposer, traits.SISO, Runnable):
+    """Selects a lattice-structured subproblem.
+
+    This decomposer requires the input state has fields ``bqm`` and 
```

Comment:
"This decomposer requires the input state has fields ... , only the keys from" -->
* "This decomposer requires that the input state have fields" or "This decomposer requires the input state to have fields" or "This decomposer requires the input state to contain fields"
* Instead of the comma splice: "``origin_embeddings``; only the ..."